### PR TITLE
EDU-111 Copy and adapt the SAML test app to educandu

### PR DESCRIPTION
**Pull request by ahelmberger**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+const TEST_WEBSITE_TUNNEL_IMAGE = 'educandu/inlets:1.0.0';
+const TEST_WEBSITE_TUNNEL_CONTAINER_NAME = 'website-tunnel';
+
+let server = null;
+Graceful.on('exit', () => {
+  server?.kill();
+});
+
+const isMac = process.platform === 'darwin';
+const containerCommandTimeoutMs = isMac ? 2000 : 1000;
```
Comment (adrianaluchian) [inline gulpfile.js:25]: 👍

Comment (ahelmberger) [reply gulpfile.js:25]: We should maybe also test this branch on one of the macs before merging.

Comment (adrianaluchian) [reply gulpfile.js:25]: Sure, let's do that together 🙏

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+~~~
+gulp
+~~~
+
+## Create a new IdP configuration
+
+Copy the certificate and entrypoint of your IdP in text files into:
+
+* `idps/<idp_name>/entrypoint.txt
+* `idps/<id_pname>/certificate.txt
```
Comment (adrianaluchian) [inline README.md]: Tiny thing: noticed reading the READMe that the quotes are not closed here on both lines

File: README.md [hunk 3]
```diff
@@ -1,2 +1,51 @@
...
+export TUNNEL_TOKEN=<your_tunnel_token>
+export TUNNEL_WEBSITE_DOMAIN=<your_tunnel_website_domain>
+~~~
+
+### Generate certificates
+
+Run the task to generate the certificates in `./tmp/<cert_name>`
+
+~~~sh
+gulp certificates
```
Comment (adrianaluchian) [inline README.md]: `gulp certificate`

File: README.md (hunk 2 above)
Comment (ahelmberger) [reply README.md]: Done, thanks

File: README.md (hunk 3 above)
Comment (ahelmberger) [reply README.md]: Done

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**